x/proof/keeper/msg_server_create_claim_test.go

```diff
@@ -8,29 +8,51 @@ import (
 	"google.golang.org/grpc/status"
 
 	keepertest "github.com/pokt-network/poktroll/testutil/keeper"
-	"github.com/pokt-network/poktroll/testutil/proof"
 	"github.com/pokt-network/poktroll/testutil/sample"
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
 	"github.com/pokt-network/poktroll/x/proof/keeper"
 	"github.com/pokt-network/poktroll/x/proof/types"
 	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
 func TestMsgServer_CreateClaim_Success(t *testing.T) {
-	appSupplierPair := proof.AppSupplierPair{
-		AppAddr:      sample.AccAddress(),
-		SupplierAddr: sample.AccAddress(),
-	}
+	proofKeeperWithDeps, ctx := keepertest.NewProofKeeperWithDeps(t)
```

Comment:
This comment might end up being moot, but I think decomposing it in one place (at the beginning) will simplify reading the code.

```go
// Decompose the proofKeeperWithDeps for easier use & readability
proofKeeper := proofKeeperWithDeps.ProofKeeper
supplierKeeper := proofKeeperWithDeps.SupplierKeeper
appKeeper := proofKeeperWithDeps.ApplicationKeeper
sessionKeeper = ...
```

Reply:
See https://github.com/pokt-network/poktroll/pull/412#discussion_r1519259812